substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2879,6 +2884,11 @@ impl<T: Config> Pallet<T> {
 		bonded_pool: BondedPool<T>,
 		reward_pool: RewardPool<T>,
 	) {
+		// The pool id of a member cannot change in any case, so we use it to make sure
+		// "member_account" is the right one.
```

Comment:
```suggestion
		// `member_account` is the right one.
```

same as in #[3110](https://github.com/paritytech/polkadot-sdk/pull/3110)